app/src/main/java/com/asfoundation/wallet/ui/iab/PaymentMethodsFragment.kt

```diff
@@ -242,17 +244,35 @@ class PaymentMethodsFragment : DaggerFragment(), PaymentMethodsView {
       payment_method_description.text = getPaymentMethodLabel(paymentMethod)
       payment_method_secondary.visibility = View.VISIBLE
       payment_method_description_single.visibility = View.GONE
+      payment_method_fee.visibility = View.GONE
       if (isBonusActive) hideBonus()
     } else {
       payment_method_description.visibility = View.VISIBLE
       payment_method_description.text = getPaymentMethodLabel(paymentMethod)
       payment_method_secondary.visibility = View.GONE
       payment_method_description_single.visibility = View.GONE
       if (isBonusActive) showBonus()
+      setupFee(paymentMethod.fee)
```

Comment:
I know that the merge is never gonna get a fee, but this along with the other stuff that is the same in both if else should be extracted